src/client/entity-editor/series-section/series-editor.tsx

```diff
@@ -0,0 +1,193 @@
+ * Field is a simple search box that allows the user to search for entities, and the Add
+ * Button allows the user to add the entity. The remove button allows to
+ * remove a entity from the list.
+ *
+ * @param {Object} props - The properties passed to the component.
+ * @param {Entity} props.baseEntity - The entity currently being edited in the
+ *        entity editor
+ * @param {Array<RelationshipType>} props.relationshipTypes - All the possible
+ *        relationship types for any pair of entities.
+ * @param {string} props.seriesType - Enity type of the series.
+ * @param {Function} props.onRemove - A function to be called when the Remove
+ *        button is clicked.
+ * @param {Function} props.onEdit - A function to be called when attribute values is
+ *        modified.
+ * @param {Function} props.onAdd - A function to be called when the add button
+ *        is clicked.
+ * @param {Array} props.seriesItemsArray - The list of entities.
+ *
+ * @returns {ReactElement} React Component as a modular section within the entity editor.
+ *
+ */
+function SeriesEditor({baseEntity, relationshipTypes, seriesType, onRemove, onAdd, onEdit, seriesItemsArray}:SeriesItemsProps) {
+	const [seriesItem, setSeriesItem] = useState(null);
+	const [targetEntity, setTargetEntity] = useState(null);
+
+	const handleEntityChange = (value: EntitySearchResult) => {
+		setTargetEntity(value);
+		// Convert "value" of type EntitySearchResult to type Entity
+		const otherEntity = {
+			bbid: _.get(value, 'id'),
+			defaultAlias: {
+				name: _.get(value, 'text')
+			},
+			disambiguation: _.get(value, 'disambiguation'),
+			type: _.get(value, 'type')
+		};
+		const relationships = generateRelationshipSelection(
+			relationshipTypes, baseEntity, otherEntity
+		);
+		if (relationships.length) {
+			// Filter out relationship type 70 - 74
+			const filterRelationships = relationships.filter(relationship =>
+				relationship.relationshipType.id > 69 && relationship.relationshipType.id < 75);
+			setSeriesItem(filterRelationships[0]);
+		}
+	};
+
+	const handleAdd = () => {
+		if (onAdd && targetEntity) {
+			// Set attributes value before adding the entity
+			seriesItem.attributes = [{attributeType: 1, value: {textValue: null}}, {attributeType: 2, value: {textValue: null}}];
+			if (seriesItem) {
+				onAdd(seriesItem);
+				// After adding the entity, clear the Entity Search Field by setting targetEntity to null.
+				setTargetEntity(null);
+			}
+		}
+	};
+
+	const handleNumberAttributeChange = (rowID, {target}) => {
+		const value = target.value === '' ? null : target.value;
+		const attributeNumber = {
+			attributeType: 2,
+			value: {textValue: value}
+		};
+		const attributePosition = {
+			attributeType: 1,
+			value: {textValue: null}
+		};
+		onEdit([attributePosition, attributeNumber], rowID);
+	};
+	return (
+		<div>
+			<h2>What {seriesType}s are part of this Series?</h2>
+			{seriesItemsArray.length ?
+				<>
+					{seriesItemsArray.map((value) => (
```

Comment:
> I think it was a great way to do it, and should be added back (is that the work you mentioned that you have locally and waiting for this PR to be merged?)

Yes indeed, I will open a different PR for it once this gets merged, and also it will be easier for you to review smaller PRs.